Dockerfile

```
@@ -0,0 +1,8 @@
+FROM node:16.1
+
+WORKDIR /mattermost-app-pagerduty
+COPY package.json .
+RUN npm install
+COPY . .
```

Optional

Just a comment that doesn't really matter for the purpose of this PR: This should also copy `package-lock.json` before `npm install`

```suggestion
COPY package.json package-lock.json .
RUN npm install
COPY . .
```